app/src/main/java/com/example/android/codeforces/Activities/ProfileActivity.java

```diff
@@ -18,6 +21,18 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_profile);
 
+        Button btn = (Button)findViewById(R.id
```

Comment:
As we are adding logout feature it should be in navigation drawer so, if we keep that on inside `profileActivity` it is hard for a user to profile section and do logout